### PR TITLE
Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by JorgeSegtowich**
Hi, Kamran.
I took the liberty of doing a translation of your roadmap in Brazilian Portuguese and I am sending a pull request to approve it who knows in a separate branch. I will also open an issue so that other Brazilians can contribute with improvements and corrections.

Thanks! ^^

### CODE REVIEW

**Comment by GabrielCoelho** [comment]
+1

**Comment by JorgeSegtowich** [comment]
Resolved

**Comment by JorgeSegtowich** [comment]
@kamranahmedse resolved

**Comment by jonaslins** [comment]
Why has this PR not been merged yet? The translation is pretty good.

File: readme.md [hunk 1]
```diff
@@ -13,47 +13,52 @@
...
-![Back-end Roadmap](./images/backend.png)
+![Back-end Roadmap](./images/pt-br/backend.png)
 
 ## DevOps Roadmap
 
-![DevOps Roadmap](./images/devops.png)
+![DevOps Roadmap](./images/pt-br/devops.png)
 
-## 🚦 Wrap Up
+## 🚦 Pra encerrar
```
Comment (blackc03r) [inline readme.md]:
```suggestion

```

File: readme.md [hunk 2]
```diff
@@ -1,52 +1,73 @@
...
-## Sponsored By	
+## Patrocinado por
 
  - [**Hackr.io** - Best Online Programming Courses & Tutorials Recommended by the Programming Community](https://hackr.io)
+ - [**Educative.io**: Become an employable Web Developer from scratch with this interactive learning track. Try a free preview today!](https://www.educative.io/track/beginning-front-end-developer)
  - [**Noon** – Develop, Document, Test and Monitor your APIs](https://noon.sh/)
  
+## Versão brasileira
+
+ - [Herbert Richers](https://www.youtube.com/watch?v=At1BlQ0EGEU)
```
Comment (Henriquepb22) [inline readme.md]:
```suggestion

```

**Comment by kamranahmedse** [comment]
Thank you. We can't merge it due to the nature of these projects. However, we are working on [the website](https://roadmap.sh) which will allow easier maintenance and introducing translations. I will reach out to you for that :)

**Pull request closed without merging**